packages/web-fragments/src/elements/reframed/main-patches.ts

```diff
@@ -0,0 +1,176 @@
+import { ReframedShadowRoot, reframedMetadataSymbol } from './reframed';
```

Comment:
We might want to split these imports into another module to avoid potential circular dependencies. Also maybe explicitly use a type import: `import { type ReframedShadowRoot }`